src/boot/efi/boot.c

```diff
@@ -518,7 +518,8 @@ static BOOLEAN menu_run(
         BOOLEAN refresh = TRUE, highlight = FALSE;
         UINTN x_start = 0, y_start = 0, y_status = 0;
         UINTN x_max, y_max;
-        CHAR16 **lines = NULL, *status = NULL, *clearline = NULL;
+        CHAR16 **lines = NULL, *status = NULL;
```

Comment:
lines can use _cleanup_(strv_free) now and status should be fine to _cleanup_ too

Reply:
i don't think we can do that. lines is not NULL terminated, hence doesn't work for strv_free()

Reply:
hmm, "status" is currently leaked, right?

Reply:
You're right. If you're feeling lazy, I'll fix both as part of my oom checking pass I intend to do soonish anyways.